src/messages/MessageElement.hpp

```diff
@@ -26,91 +26,97 @@ using ImagePtr = std::shared_ptr<Image>;
+    BttvEmoteImage = (1LL << 6),
+    BttvEmoteText = (1LL << 7),
     BttvEmote = BttvEmoteImage | BttvEmoteText,
 
-    ChannelPointReward = (1 << 8),
+    ChannelPointReward = (1LL << 8),
     ChannelPointRewardImage = ChannelPointReward | TwitchEmoteImage,
 
-    FfzEmoteImage = (1 << 10),
-    FfzEmoteText = (1 << 11),
+    FfzEmoteImage = (1LL << 10),
+    FfzEmoteText = (1LL << 11),
     FfzEmote = FfzEmoteImage | FfzEmoteText,
     EmoteImages = TwitchEmoteImage | BttvEmoteImage | FfzEmoteImage,
     EmoteText = TwitchEmoteText | BttvEmoteText | FfzEmoteText,
 
-    BitsStatic = (1 << 12),
-    BitsAnimated = (1 << 13),
+    BitsStatic = (1LL << 12),
+    BitsAnimated = (1LL << 13),
 
     // Slot 1: Twitch
     // - Staff badge
     // - Admin badge
     // - Global Moderator badge
-    BadgeGlobalAuthority = (1 << 14),
+    BadgeGlobalAuthority = (1LL << 14),
 
     // Slot 2: Twitch
     // - Moderator badge
     // - Broadcaster badge
-    BadgeChannelAuthority = (1 << 15),
+    BadgeChannelAuthority = (1LL << 15),
 
     // Slot 3: Twitch
     // - Subscription badges
-    BadgeSubscription = (1 << 16),
+    BadgeSubscription = (1LL << 16),
 
     // Slot 4: Twitch
     // - Turbo badge
     // - Prime badge
     // - Bit badges
     // - Game badges
-    BadgeVanity = (1 << 17),
+    BadgeVanity = (1LL << 17),
 
     // Slot 5: Chatterino
     // - Chatterino developer badge
     // - Chatterino donator badge
     // - Chatterino top donator badge
-    BadgeChatterino = (1 << 18),
+    BadgeChatterino = (1LL << 18),
+
+    // Slot 6: FrankerFaceZ
+    // - FFZ developer badge
+    // - FFZ bot badge
+    // - FFZ donator badge
+    BadgeFfz = (1LL << 32),
 
     Badges = BadgeGlobalAuthority | BadgeChannelAuthority | BadgeSubscription |
-             BadgeVanity | BadgeChatterino,
+             BadgeVanity | BadgeChatterino | BadgeFfz,
 
-    ChannelName = (1 << 19),
+    ChannelName = (1LL << 19),
 
-    BitsAmount = (1 << 20),
+    BitsAmount = (1LL << 20),
 
-    ModeratorTools = (1 << 21),
+    ModeratorTools = (1LL << 21),
 
-    EmojiImage = (1 << 23),
-    EmojiText = (1 << 24),
+    EmojiImage = (1LL << 23),
+    EmojiText = (1LL << 24),
     EmojiAll = EmojiImage | EmojiText,
 
-    AlwaysShow = (1 << 25),
+    AlwaysShow = (1LL << 25),
 
     // used in the ChannelView class to make the collapse buttons visible if
     // needed
-    Collapsed = (1 << 26),
+    Collapsed = (1LL << 26),
 
     // used for dynamic bold usernames
-    BoldUsername = (1 << 27),
-    NonBoldUsername = (1 << 28),
+    BoldUsername = (1LL << 27),
+    NonBoldUsername = (1LL << 28),
 
     // for links
-    LowercaseLink = (1 << 29),
-    OriginalLink = (1 << 30),
+    LowercaseLink = (1LL << 29),
+    OriginalLink = (1LL << 30),
 
     // ZeroWidthEmotes are emotes that are supposed to overlay over any pre-existing emotes
     // e.g. BTTV's SoSnowy during christmas season
-    ZeroWidthEmote = (1 << 31),
+    ZeroWidthEmote = (1LL << 31),
 
```

Comment:
I suggest leaving a comment here stating that bit 32 is already used by a flag.
```suggestion

// (1LL << 32) is being used by `BadgeFfz`, see above

```